terraso_backend/apps/auth/services.py

```diff
@@ -48,8 +56,22 @@ def _persist_user(self, email, first_name="", last_name=""):
         if update_name:
             user.save()
 
+        user_id = str(user.id)
+        has_picture = profile_image_service.exists(user_id)
+        if not has_picture and picture_file:
+            image_file = ContentFile(picture_file)
+            profile_image_service.upload(user_id, image_file)
+
         return user
 
+    def _get_profile_image(self, url):
```

Comment:
Can we move all this file handling to the storage app? We could have something like `ProfileImaageService().upload_from_url(user_id, remote_url)`. I miss some exceptions handling, and it's going to be better to hide these complexities under storage app. Auth shouldn't be dealing with files.

Reply:
@cacarrara This is a particular process related to how google is sending the picture data, this is not related to the `storage` app, I think it is better to leave this here related to the auth providers, maybe I can change the name to `_get_google_profile_image`

Reply:
I'm not referring to the specific process of taking a Google image. I'm referring to the generic process of getting a file from a remote location and putting into our storage. I really don't think it's appropriate to make file handling on authentication app. This breaks the single responsibility principle.

Reply:
That makes more sense to me, I think that can fall under the storage responsabilities because it can upload a url resrouce to a file, I can change that